Source/stores.cpp

```diff
@@ -75,6 +78,11 @@ void InitStores()
 	for (i = 0; i < SMITH_PREMIUM_ITEMS; i++)
 		premiumitem[i]._itype = ITYPE_NONE;
 
+	for (i = 0; i < SMITH_ITEMS; i++) {
+		smithbuybackitems[i]._itype = ITYPE_NONE;

```

Comment:
use WITCH_ITEMS for witch items

Reply:
Sorry I thought you asked me to do this and pushed this - now I realize you had done the changes already.